Automattic/wp-calypso
# Dev Survey: Ensure Consistent Colour Scheming

Fixes #94014

## Proposed Changes

* Ensures that the colour scheming on for the Dev Survey is consistent with the rest of Calypso by using the correct colour variable. 

## Why are these changes being made?
The colours currently don't match the rest of the UI because they have been hardcoded using Core Gutenberg colours (whereas they should be using Blueberry colours). 

## Testing Instructions

Navigate to `/me/developer` and verify that the colours and buttons appear correctly. 

| Before | After |
|--------|--------|
| <img width="1196" alt="Screenshot 2024-09-02 at 12 33 25" src="https://github.com/user-attachments/assets/b57d19f9-da89-43e6-9df1-4810a066ad79"> | <img width="1182" alt="Screenshot 2024-09-02 at 12 33 12" src="https://github.com/user-attachments/assets/8b055eb7-2ad9-48c4-801a-018ed1fe6e46"> |

cc @taipeicoder, @nightnei, @wongasy
